docs/configuration/node-config.md

```diff
@@ -65,6 +65,8 @@ This section contains the configuration options for a Searcher.
 
 | Property | Description | Default value |
 | --- | --- | --- |
+| `aggregation_memory_limit` | Controls the maximum amount of memory that can be used for aggregations before aborting. This limit is shared among all queries on splits on a node for a single query. It is used to prevent excessive memory usage during the aggregation phase, which can lead to performance degradation or crashes. | `500M`|
+| `aggregation_bucket_limit` | Determines the maximum number of buckets returned to the client. | `65000` |
```

Comment:
does it abort if the maximum number of buckets exceeds the limit?

Reply:
The check is done after the aggregations, before returning to the client